.gitignore

```diff
@@ -111,14 +111,11 @@ venv*
 # mypy
 .mypy_cache/
 
```

Comment:
No need to change this file. You can use `git add data -f ` to forcely add the changes if it's necessary.

Reply:
ok